src/logger.ts

```diff
@@ -161,6 +161,12 @@ function createLogMethod(level: LogLevel): LogMethod {
         }
         if (arguments.length) {
             const record = createLogRecord.apply(this, arguments);
+
+            // If no log handlers are registered, default to consoleHandler.
+            if (logHandlers.length === 0) {
+                registerLogHandler(consoleHandler);
```

Comment:
I prefer not to register a handler at this point. If should be possible to register a handler after the first log statement without having to remove the consoleHandler. So simply *use* the `consoleHandler`, but do not register it.